pkgs/by-name/kr/krep/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krep";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "davidesantangelo";
+    repo = "krep";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-KN/vQlpRt0w/NZZeUUqHKpSkzv77gx6ZzwCWTPFo8IA=";
+  };
+
+  makeFlags = [
+    "CC=${stdenv.cc.targetPrefix}cc"
+    "ENABLE_ARCH_DETECTION=0"
+  ];
+
+  installFlags = [
+    "PREFIX=${placeholder "out"}"
+  ];
+
```

```suggestion
  doCheck = true;
```
There is a test suite we should run.